tests/unittests/utils/test_radio.py

```
@@ -0,0 +1,1219 @@
+import torch
+import traceback
+import tempfile
+import os
+import sys
+
+PASSED = 0
+FAILED = 0
+
+def test(name):
+    """Decorator for test functions."""
+    def decorator(func):
```

Ruff (0.14.14)

[warning] Missing return type annotation for private function `decorator`

(ANN202)